test/DotNetty.Handlers.Proxy.Tests/ProxyHandlerTest.cs

```diff
@@ -86,7 +86,7 @@ public ProxyHandlerTest(ITestOutputHelper output) : base(output)
             ClearServerExceptions();
         }
         
-        [Theory(Timeout = 5000)]
+        [Theory(Timeout = 5000, Skip = "This test is not reliable and skipped for now. See https://msazure.visualstudio.com/One/_workitems/edit/25095979")]
```

Comment:
Try to see if we can disable them only on Ubuntu

Reply:
@tomkerkhove is it possible?

Reply:
We can try to automatically identify the OS it runs on and skip that way. What do you think about that?

Reply:
sure